Source/gg2/Objects/Weapons/Medigun.events/User Event 1.xml

```diff
@@ -16,12 +16,32 @@
         <argument kind="STRING">{
     if(!instance_exists(healTarget) and readyToShoot)
     {
-        var bestTarget, bestTargetDist;
+        var bestTarget, bestTargetDist, testx, testy;
         bestTarget=noone;
-        bestTargetDist=maxHealDistance;
+        bestTargetDist = maxHealDistance;
+        
+        testx = round(owner.x) + cos(degtorad(owner.aimDirection))*owner.aimDistance;
```

Comment:
Why round here? It doesn't make much of a difference, but I think it's wrong, and less code is better.